### ISSUE TITLE
will it work with webpack 5?

### ISSUE THREAD

**Issue by tukab** [faceyspacey/react-universal-component#222]
will it work with webpack 5?

**Comment by cdoublev** [comment]
Yes, but if you're using it, `babel-plugin-universal-import` requires [this small change](https://github.com/faceyspacey/babel-plugin-universal-import/pull/143).
